### PR TITLE
Icu 15743 tag workflow

### PR DESCRIPTION
**Pull request by cameronperera**
# Description

This PR adds the workflow for creating a tag specific worker filter

## Screenshots (if appropriate)

https://github.com/user-attachments/assets/68185c0d-b113-47e8-ad75-e43e6a3e0f74

## How to Test

1. Go to the workers tab on a Target, toggle the "show generator" toggle
2. Enter in the key and value
3. Should be able to copy the formatted result

## Checklist

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: addons/core/translations/en-us.yaml [hunk 1]
```diff
@@ -141,6 +141,13 @@ worker-filter-generator:
     title: Filter generator
     description: Choose what you want to format into a filter.
     link: Learn more about formatting a worker filter.
+    input-values:
+      title: Input values
+      description: Next, input what you want to format
```
Comment (lisbet-alvarez) [inline addons/core/translations/en-us.yaml:146]:
```suggestion
      description: Next, input what you want to format.
```

File: addons/core/translations/en-us.yaml [hunk 2]
```diff
@@ -141,6 +141,13 @@ worker-filter-generator:
...
+    formatted-result:
+      title: Formatted result
+      description: Copy and paste this into the editor above
```
Comment (lisbet-alvarez) [inline addons/core/translations/en-us.yaml:149]:
```suggestion
      description: Copy and paste this into the editor above.
```

File: ui/admin/app/components/worker-filter/index.hbs [hunk 3]
```diff
@@ -73,6 +77,63 @@
...
+            'worker-filter-generator.filter-generator.input-values.title'
+          }}</F.Legend>
+        <F.HelperText>{{t
+            'worker-filter-generator.filter-generator.input-values.description'
+          }}</F.HelperText>
+        <F.Control>
+          <Hds::Form::TextInput::Field
+            @value={{this.tagKey}}
+            name='tag_key'
+            @width='320px'
```
Comment (lisbet-alvarez) [inline ui/admin/app/components/worker-filter/index.hbs:93]: im wondering if we should leave this as pixels or use rem instead. Both work, just wondering what the convention is.

File: ui/admin/app/components/worker-filter/index.hbs [hunk 4]
```diff
@@ -73,6 +77,63 @@
...
+          'worker-filter-generator.filter-generator.formatted-result.description'
+        }}</F.HelperText>
+      <F.Control>
+        <Hds::Text::Body @tag='p' @weight='semibold'>{{t
+            'worker-filter-generator.filter-generator.formatted-result.label'
+          }}</Hds::Text::Body>
+        <div class='generated-results-container'>
+          <Hds::Form::TextInput::Base
+            readonly
+            id='generatedValue'
```
Comment (lisbet-alvarez) [inline ui/admin/app/components/worker-filter/index.hbs:128]: i think for id values we usually use dashes `id='generated-value'`. Does that work?

File: ui/admin/app/components/worker-filter/index.hbs (hunk 3 above)
Comment (cameronperera) [reply ui/admin/app/components/worker-filter/index.hbs:93]: I looked around and we primarily use `%` for the `@width` property. The issue with that is the input would get much smaller if the user wants change the width of the screen for split screen for example. I don't think `rem` would give us much of a change. I would lean towards leaving it as is.

File: ui/admin/app/components/worker-filter/index.hbs (hunk 4 above)
Comment (cameronperera) [reply ui/admin/app/components/worker-filter/index.hbs:128]: I will test that now.

Comment (cameronperera) [reply ui/admin/app/components/worker-filter/index.hbs]: It worked, updated.

**Review by lisbet-alvarez — APPROVED** [review]
🥳

**Pull request merged**